miscellaneous/toggle-rotation-lock.ahk

```diff
@@ -0,0 +1,37 @@
+#NoTrayIcon
+#SingleInstance Force
+#NoEnv
+#Include %A_ScriptDir%\..\workspaces\utils.ahk
+
+; This script toggles the Windows 10 'rotation lock' feature
+
+; Run as administrator
+; Required to modify registry keys
+full_command_line := DllCall("GetCommandLine", "str")
+if not (A_IsAdmin or RegExMatch(full_command_line, " /restart(?!\S)"))
+    try
+        Run *RunAs "%A_AhkPath%" /restart "%A_ScriptFullPath%"
```

Comment:
This would fix https://github.com/lu0/autohotkey-config/issues/7 if put on every script in this repo, although I still need to find a way to run all these scripts as Admin an Startup without Windows asking to confirm every time (+10 times).